.gitmodules

```diff
@@ -82,3 +82,9 @@
 [submodule "src/cf-sle12-release"]
 	path = src/cf-sle12-release
 	url = https://github.com/SUSE/cf-sle12-release.git
+[submodule "src/cephfs-bosh-release"]
+	path = src/cephfs-bosh-release
```

Comment:
We _could_ just make the submodule be called `src/cephfs-release` even though it doesn't match the repo name…